src/dev/build/tasks/os_packages/docker_generator/run.ts

```diff
@@ -131,13 +131,6 @@ export async function runDockerGenerator(
     dockerBuildDir
   );
 
-  if (flags.ironbank) {
```

Comment:
This wasn't used, and cpy throws errors if nothing matches.  I think that's a reasonable API change, but can catch it if there's any concerns.